quixstreams/app.py

```diff
@@ -746,6 +1087,17 @@ def run(self, dataframe: Optional[StreamingDataFrame] = None):
                 "the argument should be removed.",
                 FutureWarning,
             )
+        if not self._dataframe_registry.consumer_topics:
+            if count or timeout_starts_on_first_message:
```

Comment:
Did you mean `timeout` here?
```suggestion
            if timeout or timeout_starts_on_first_message:
```

Reply:
It should be correct: `count` is not a valid stop option when it's just a plain source (no sdf). I have added a comment to clarify what check is going on here =)